pkg/protocol/http1/client_unix_test.go

```diff
@@ -32,9 +32,9 @@ import (
 
 func TestGcBodyStream(t *testing.T) {
 	srv := &http.Server{Addr: ":11001", Handler: http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		w.Write([]byte("hello\n"))
-		time.Sleep(time.Second)
-		w.Write([]byte("world\n"))
+		for range [1024]int{} {
```

Comment:
What is the purpose of this range?

Reply:
Send a large amount of body to make the issue more pronounced.